Lib/test/test_dis.py

```diff
@@ -1050,12 +1105,12 @@ def test_explicit_first_line(self):
         self.assertEqual(list(actual), expected_opinfo_outer)
 
     def test_source_line_in_disassembly(self):
-        # Use the line in the source code
-        actual = dis.Bytecode(simple).dis()[:3]
+        # Use the line in the source code (split extracts the line no)
+        actual = dis.Bytecode(simple).dis().split(" ")[0]
```

Comment:
You're right--I hadn't noticed that there's leading whitespace below 3 digits, which breaks both of the string method solutions. (And now I understand the `{:>3}`).

For the sake of clarity, I'll go with:

~~~~
actual = dis.Bytecode(simple).dis().lstrip(" ").partition(" ")[0]
expected = str(simple.__code__.co_firstlineno)
~~~~

unless anyone notices any problems with that. (And I'll do that in a separate commit, since we're fixing a bug caught by the actual patch to `.dis()`).